stagemonitor-requestmonitor/src/main/java/org/stagemonitor/requestmonitor/tracing/NoopSpan.java

```diff
@@ -0,0 +1,81 @@
+package org.stagemonitor.requestmonitor.tracing;
+
+import java.util.Collections;
+import java.util.Map;
+
+import io.opentracing.Span;
+import io.opentracing.SpanContext;
+
+public class NoopSpan implements Span {
```

Comment:
It used to be final but I use `spy(NoopSpan.INSTANCE)` in `MockTracer` to be able to `verify` that certain methods have been called (for example `verify(span).setTag("exception.message", "bazinga!")`). Mockito can't `spy` final classes.